examples/user_guide/Deploy_and_Export.ipynb

```diff
@@ -129,6 +129,47 @@
     "The app will now run on a Bokeh server instance separate from the Jupyter notebook kernel, allowing you to quickly test that all the functionality of your app works both in a notebook and in a server context."
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### ipywidgets\n",
+    "\n",
+    "If the `jupyter_bokeh` package is installed it is also possible to render Panel objects as an ipywidget, either by enabling it globally using:\n",
```

Comment:
```suggestion
    "If the `jupyter_bokeh` package is installed it is also possible to render Panel objects as an ipywidget rather than using Bokeh's internal communication mechanisms.  You can enable ipywidgets support  globally using:\n",
```